weather-app/src/app/landing-page/landing-page.component.ts

```diff
@@ -1,22 +1,42 @@
 import { Component, OnInit } from '@angular/core';
+import { byCountry } from 'country-code-lookup';
+import { WeatherService } from '../services/weather.service';
 
 @Component({
   selector: 'app-landing-page',
   templateUrl: './landing-page.component.html',
   styleUrls: ['./landing-page.component.scss', '../../styles/_container.scss']
 })
 export class LandingPageComponent implements OnInit {
+  public currentCity: string = '';
+  public currentCountry: string = '';
+  public currentDay: string = '';
+  public currentTime: string = ''; 
+  public currentDegrees: string = ''; 
+  public curentWeatherIconUrl = '';
+ 
   isDarkMode = false;
-  currentCity = 'Ivano-Frankivsk';
-  currentCountry = 'UA';
-  currentDay = 'Monday';
-  currentTime = '06:00';
-  currentDegrees = '+12';
-  curentWeatherIconUrl = 'https://raw.githubusercontent.com/basmilius/weather-icons/029d142b34871bcbc90d7cd46081c50310c831c5/production/fill/svg/overcast-day-fog.svg';
-  
-  constructor() {
+  weekday = ["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"];
+
+  constructor(private weatherService: WeatherService) {
   }
 
   ngOnInit(): void {
+    this.weatherService.getCurrentWeather('Lviv').subscribe((data: any) => {
```

Comment:
Are you going to use a variable instead of 'Lviv'? Might be better to prepare a variable for this - `defaultCity`, which is going to be used if there's no `currentLocationCoords` from `navigator.geolocation.getCurrentPosition`

Reply:
Fixed.